driver/manager.go

```diff
@@ -178,3 +181,34 @@ func (d *DriverHandle) HistoryAPISupported(ctx context.Context) bool {
 	})
 	return d.historyAPISupported
 }
+
+func (d *DriverHandle) HostGatewayIP(ctx context.Context) (net.IP, error) {
+	d.hostGatewayIPOnce.Do(func() {
+		if !d.Driver.IsMobyDriver() {
+			return
+		}
+		c, err := d.Client(ctx)
+		if err != nil {
+			d.hostGatewayIPErr = err
+			return
+		}
+		workers, err := c.ListWorkers(ctx)
+		if err != nil {
+			d.hostGatewayIPErr = err
+			return
+		}
+		if len(workers) == 0 {
+			return
+		}
+		// TODO: Use HostGatewayIP const from github.com/docker/docker/builder/builder-next/worker/label pkg
+		if v, ok := workers[0].Labels["org.mobyproject.buildkit.worker.moby.host-gateway-ip"]; ok && v != "" {
+			ip := net.ParseIP(v)
+			if ip == nil {
+				d.hostGatewayIPErr = errors.Wrapf(err, "failed to parse host gateway IP: %s", v)
```

Comment:
This will always be `nil`, because `errors.Wrap()` returns `nil` if `err` is `nil`

Reply:
:man_facepalming: